packages/core/src/constants.ts

```diff
@@ -4,6 +4,17 @@ import envPaths from 'env-paths'
 
 export const DEFAULT_KUAI_PRIVATE_KEY = '0xfd686a48908e8caf97723578bf85f746e1e1d8956cb132f6a2e92e7234a2a245'
 
+export const CKB_CLI_RELEASE_URL = 'https://github.com/nervosnetwork/ckb-cli/releases/download/'
+export const DEFAULT_CKB_CLI_VERSION = '1.4.0'
+export const DEFAULT_BIN_PATH = '.bin'
+
+export const DEFAULT_DEVNET_ACCOUNT_PKS = [
+  '0xfd686a48908e8caf97723578bf85f746e1e1d8956cb132f6a2e92e7234a2a245',
+  '0x5368b818f59570b5bc078a6a564f098a191dcb8938d95c413be5065fd6c42d32',
+  '0xd6013cd867d286ef84cc300ac6546013837df2b06c9f53c83b4c33c2417f6a07',
+  '0x13b08bb054d5dd04013156dced8ba2ce4d8cc5973e10d905a228ea1abc267e60',
```

Comment:
These are default built-in private keys from random generation, it will be used as genesis_issued_cells when devnet is created

Reply:
Should they be loaded during runtime or injected on installation if they are generated randomly?

Reply:
Any idea on this?

Reply:
When generating some pk here, these pk need to be injected  into devnet, and saved in some place, currently kuai has no place to save temporary data, it feels like there might be a `.kuai` folder or somewhere to save this temporary data
Do you think this will be better?

Reply:
How about using what I used for deploying scripts?

https://github.com/ckb-js/kuai/blob/962d503e75dc808812ec216409877ddc8545c109/packages/core/src/helper.ts#L30

Reply:
Looks good

Reply:
I've found a preset `_cache` directory in the project template https://github.com/ckb-js/kuai/tree/develop/packages/core/sample-projects/_cache

It's properly prior to the shared cache directory